skype-sender-exercise/skype_conf.py

```diff
@@ -0,0 +1,10 @@
+"""
+Message and necessary configuration information
+"""
+
+MESSAGE = "This is a message from Drishya"
```

Comment:
Do we need to keep the message configurable? I feel this is very much tied to the test case, and we may never use this message ever other than in that test case. 

Code looks good.

Reply:
If any change to the message , It shouldn't impact the actual test code, thats the reason, provided in config. All the vulnerable or variables, that need to change in one place, thats the thought behind this . Thanks for your comment